packages/adyen-integration/src/index.ts

```diff
@@ -0,0 +1,6 @@
+export { default as AdyenV2PaymentMethod } from '../src/adyenv2/AdyenV2PaymentMethod';
+export { default as AdyenV2Form } from '../src/adyenv2/AdyenV2Form';
+export { default as AdyenV2CardValidation } from '../src/adyenv2/AdyenV2CardValidation';
```

Comment:
Why do making export `AdyenV2Form`, `AdyenV2CardValidation`, `AdyenV3Form` and `AdyenV3CardValidation` components in this file?